### DISCUSSION TITLE
Outreachy Internship

### DISCUSSION THREAD

**Discussion by ummyhajj** [MDAnalysis/mdanalysis#4487] [Introductions]
Hello everyone, I am Ummi Hajaratu Siba, an Outreachy applicant intrerested in contributing to the MDAnalysis project; "Develop a communication strategy for a growing MDAnalysis user contributor base".

**Comment by adetutudeborah** [comment]
Hello everyone. My name is Adetutu Oluwasanmi, a microbiology graduate with a background in scientific research and communication. I am excited about the opportunity to contribute significantly to the expansion of MDAnalysis' contributor base and its outreach to the target audience.

**Reply by SiziweLupondo** [reply]
Hello Edu , I personally cant wait to see how youll bring about your current knowladge to this project .

PS:  Welcome to the party 😊

**Comment by Dehbeie** [comment]
Hello, My name is Dehbeie, and I'm excited to introduce myself as a data analyst interested in joining MDAnalysis. I am impressed by the incredible work you all do in the field of molecular dynamics.

I have a strong background in data analysis, with experience in in scientific research and very great communication skill. I'm particularly passionate about applying my skills to the field of molecular dynamics and contributing to the development of MDAnalysis.

I would love to learn more on how to contribute to the team. I would be thrilled to be a part of this team.

Thank you for considering my introduction, and I look forward to the possibility of working together with the MDAnalysis team.

**Reply by SiziweLupondo** [reply]
Hello Deh , Welcome.

I have done some  data / business analysis as well. i guess thats what we have in common otherwise i cannot wait to learn from you

**Reply by Dehbeie** [reply]
Lol....I can't wait to learn from you too 🤗

**Comment by the-Kings-daughter** [comment]
Hello Everyone!
My name is Faith Ekechi Onyinyechi. I am a graduate in the field of Electrical/Electronic Engineering residing in Cross River State, Nigeria. 
My primary focus will be on the MD analysis project 1 which involves developing a communications strategy for a growing MDAnalysis user and contributor base. Although I am relatively new to open source, I am fully committed to learning and putting in extra effort to address any gaps in my knowledge.
I am excited about the opportunity to gain valuable insights from my esteemed mentors and fellow applicants.

**Reply by SiziweLupondo** [reply]
Hello Faith , Its good to have you here. 
I hardly meet Electric guys in these spaces, are you transitioning to tech or you are already within.

Hopefully i will learn a great deal from you.

**Reply by the-Kings-daughter** [reply]
lol. I'm not a guy. I'm a lady and I have been in tech since early last year. I'm sure to learn a lot from you and other applicants as well.

**Comment by SiziweLupondo** [comment]
Hello everyone, my name is Siziwe Lupondo, From South Africa. I have BSc Computer science and currently looking to do Masters in Data Science. 
I am so happy to have made it this far. 
I am looking forward to contribute to ''develop a communications strategy for a growing MDAnalysis user and contributor base"

I spend most of my time coding , listening to podcasts abouts artificial intelligence, Business Intelligence and Data stories. 
I cant wait to spend the next few weeks learning and coding with all of you here.

Cheers to taking this journey together , learning and to meaningful contributions.🎆💃🎊🎉

**Reply by ummyhajj** [reply]
You are welcome Siziwe, I also have a Computer Science background and looking forward to pursue a postgraduate degree in Data Science. We share that in common.
Nice to meet you and looking forward to learning and sharing knowledge 📕 💐📌🇬🇭

**Comment by ummyhajj** [comment]
Great!!!💐

**Comment by ummyhajj** [comment]
Are you guys following the conversation on https://discord.com/channels/807348386012987462/958779313551986699? Get onboard and lets take a deep dive into the world 🌍of MDAnalysis😇💐

**Comment by victorezealuma** [comment]
Hello Everyone,

My name is Victor, an outreachy contributor. I am a data analyst with a degree in software engineering. 
I am excited to be contributing to the enhancement of MDAnalysis training materials.

**Comment by Diksha192001** [comment]
Hi everyone!
My name is Diksha. I am a B.E Computer Science student. I am glad to be here and very excited to contribute to the second project :)

**Comment by ikehdarlington** [comment]
Hello MDAnalysis Community! I am a budding data analyst with a degree in Mechanical Engineering. I am delighted to join the MDAnalysis community and contribute to the project's growth and development. I want to improve my skills in technical documentations, communication, research and data analysis while contributing to the project.

**Comment by Blessing-Mba** [comment]
Hello MDAnalysis Community! ..My name is Blessing Mba and I'm excited to be here. I'm interested in contributing to enhancing installation and training materials to support accessible MDAnalysis workshops project, I'm open to learning and expanding my knowledge here

**Reply by SiziweLupondo** [reply]
Hello @Blessing-Mba , Welcome

**Comment by Nikilight** [comment]
Hello Everyone. I am Adenike Oladimeji-Kasumu, A Data Analyst with a background in Microbiology. I'm excited to be in the MDAnalysis community. 
Please, I would appreciate if anyone can guide me on how to start my contribution here. Thank you.

**Reply by SiziweLupondo** [reply]
Hello @Nikilight , Please join the discord channel for more information

**Reply by Nikilight** [reply]
Please how do i join the discord community. I have been trying to find my way around here. Help please

**Reply by jennaswa** [reply]
@Nikilight - Please read the [participating](https://www.mdanalysis.org/#participating) section on the MDAnalysis website to obtain the invitation link for joining Discord. You can then visit the #outreachy channel for relevant discussions on Discord.

**Reply by Nikilight** [reply]
Thank you @jennaswa I have been able to join Discord. I am currently going through the information here to see how I can start contributing.

**Comment by ummyhajj** [comment]
You are all welcome😇👋

**Comment by amarasparkle** [comment]
Hello Everyone!
My name is Amarachi Collins-Nnaozie. I am a Dental therapist graduate and a UI/UX designer. 
I'm excited to be a contributor to MD analysis project 1 which involves developing a communications strategy for a growing MDAnalysis user and contributor base. 
I am quite new to open source but fully committed to learning and putting in my best effort to any given task. 
I look forward to gaining valuable insights  the mentors and fellow applicants alike.

**Reply by amarasparkle** [reply]
Please i would appreciate guidance on how to begin contribution

**Reply by ummyhajj** [reply]
Warmest welcome @amarasparkle

**Reply by jennaswa** [reply]
@amarasparkle - For more details and tips on how to get started with your contribution, please visit the [MDAnalysis Outreachy wiki page](https://github.com/MDAnalysis/mdanalysis/wiki/Outreachy).

**Comment by ummyhajj** [comment]
Kindly clink on the link https://discord.com/channels/807348386012987462/958779313551986699 to access the Outreachy channel on discord. 
It's also imperative to contribute to discussions here.

**Reply by amarasparkle** [reply]
<img width="1351" alt="Screenshot 2024-03-11 at 17 15 55" src="https://github.com/MDAnalysis/mdanalysis/assets/162321762/68e81603-6cb7-49d8-ac83-da88b9418590">
This is what i'm seeing on discord. how do i gain access?

**Reply by amarasparkle** [reply]
I have access now. thanks

**Comment by ummyhajj** [comment]
If you still need further clarification, please don't hesitate to reach out. We are here to help😇👌

**Comment by ighaloirene** [comment]
Hi Everyone, My name is Irene Ighalo. Looking forward to contributing to this Project. Can anyone help with discord, i don't seem to have access to the discord community. It is blank. Thank you

**Reply by jennaswa** [reply]
@ighaloirene - Please read the [participating](https://www.mdanalysis.org/#participating) section on the MDAnalysis website to obtain the invitation link for joining Discord. You can then visit the #outreachy channel for relevant discussions on Discord.

If that doesn't work, please provide additional details/links about how you have tried to access Discord so we can help you troubleshoot.

**Reply by ighaloirene** [reply]
Okay thank you for your response. I'll do just that @jennaswa

**Comment by maha-sachin** [comment]
Hi I am Maha from canada and outreachy intern. I'm excited to be a contributor to MD analysis project 1 which involves developing a communications strategy for a growing MDAnalysis user and contributor base.

**Comment by PreciousChikwendu** [comment]
I am Precious Chikwendu, and I'm excited to be a part of this community as we embark on our journey with Outreachy Internship. My project focuses on contributing to MD Analysis contribuion and also looking forward to learning and growing together with all of you.

**Comment by deveshidwivedi** [comment]
Hello everyone! I'm Deveshi Dwivedi from India, currently pursuing a Bachelor's in Technology with a major in Computer Science. I'm super glad to be here!

**Comment by Sneha6003** [comment]
Hello community! I am Sneha Singh from India, pursuing Computer Science in Engineering. looking forward to contribute and collaborate with you all!

**Comment by ifylanky** [comment]
Hi everyone,
My name is Ifeoma Aniekwe from Nigeria, I hold a bachelor's degree in sociology and anthropology, and a master's degree in organizational behavior. My professional journey includes training as a product manager and completing Google digital marketing and ecommerce courses. I have hands-on experience creating engaging content for social media platforms such as Instagram, LinkedIn, and Twitter.
I am applying to the outreachy application and working on the project "Develop a communications strategy for a growing MDAnalysis user and contributor base" 
I will appreciate all the guide to help submit a successful application and get selected as an intern. I am also yet to have any communication with my mentor please I need more information on how to reach my mentor.

**Reply by amarasparkle** [reply]
Hi Ifylanky welcome onboard. you can join the discord group and introduce yourself

**Reply by ifylanky** [reply]
Ok, Thank you Amara

**Comment by PreciousChikwendu** [comment]
You're welcome dear, don't forget to check the pinned message for amazing
updates.

**Reply by ifylanky** [reply]
OK, thank you

**Comment by LynnTiampati001** [comment]
Hello everyone, my name is  Lynn Tiampati from Kenya. I'm thrilled to be part of the MDAnalysis community! I'm obviously very late to the party but here we are. I have a strong background in Statistics and programming, and I look forward to contributing to the development of a robust communications strategy for our growing MDAnalysis user and contributor base. 
As we navigate the exciting journey ahead, I look forward to collaborating with all of you to enhance our community's reach, impact, and accessibility. Together, let's pave the way for a brighter future of MDAnalysis!
I just submitted the pre-proposal form and would like to know how to contribute further to this project, especially concerning the contribution URL.

**Comment by ummyhajj** [comment]
Kindly join the discord channel for further guidelines.
